app/src/androidTest/java/com/android/unio/components/event/EventDetailsTest.kt

```diff
@@ -296,9 +301,16 @@ class EventDetailsTest : TearDown() {
   fun testEventDetailsData() {
     val event = events[1]
     setEventScreen(event)
-    composeTestRule.onNodeWithText(event.title).assertDisplayComponentInScroll()
-    composeTestRule.onNodeWithText(event.description).assertDisplayComponentInScroll()
-    composeTestRule.onNodeWithText(event.location.name).assertDisplayComponentInScroll()
+    composeTestRule.onNodeWithText(event.title, substring = true).assertDisplayComponentInScroll()
+    composeTestRule
+        .onNodeWithText(event.description, substring = true)
+        .assertDisplayComponentInScroll()
+
+    Espresso.onView(ViewMatchers.isRoot()).perform(ViewActions.swipeUp())
```

Comment:
Amazing tests! Using Expresso, thinking about me every time oh, is a great way to test this functionality !